dtm0/fop.c

```diff
@@ -108,10 +108,16 @@ M0_INTERNAL void m0_dtm0_fop_fini(void)
 }
 
 enum {
```

Comment:
Have UTs and ./all2all passed?

Reply:
Existing UT has passed. I will run all2all test and update.

Reply:
I have updated the UT for dtm0 client to send and validate persistent messages. I have tested with these UT changes.